pkg/operator/operands/prometheus/resources.go

```diff
@@ -0,0 +1,122 @@
+// Copyright 2025 NVIDIA CORPORATION
+// SPDX-License-Identifier: Apache-2.0
+
+package prometheus
+
+import (
+	"context"
+
+	monitoringv1 "github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/log"
+
+	kaiv1 "github.com/NVIDIA/KAI-scheduler/pkg/apis/kai/v1"
+)
+
+const (
+	mainResourceName = "prometheus"
```

Comment:
```suggestion
	mainResourceName = "kai"
```
This will result in "prometheus-kai" (or kai-prometheus, I don't remember)
if you leave it as "prometheus", it will be "prometheus-prometheus"